day_two/index.js

```diff
@@ -0,0 +1,71 @@
+const testData = require("./test_input");
+
+const problemOne = (noun = 12, verb = 2) => {
+  const fixedOpCode = [testData[0], noun, verb, ...testData.slice(3)];
+  const opCodeResult = executeIntcode(fixedOpCode);
+  return opCodeResult[0];
+};
+
+const problemTwo = () => {
+  let noun = getRandomInt(99);
```

Comment:
interesting move, another way to do this would be to have a nested for loop, where the outer loop iterates once for each inner full completion of the entire range

Reply:
Yeah, I didn't realize at first that there was an upward bound of 99. I think nested loops would make more sense here, and be more idiomatic.